components/CartSidebar.vue

```diff
@@ -0,0 +1,150 @@
+<template lang="pug">
+  div.cart-sidebar-wrapper(:class="openSidebarCart ? 'active' : ''")
+    div.overlay-cart-sidebar(@click="closeSidebarCart()")
+    aside.cart-sidebar
+        div.cart-sidebar-header
+            h4.title.is-3 Cart
+            i.fa.fa-times.fa-lg(@click="closeSidebarCart()")
+        div.cart-sidebar-body
+          div(v-if="total > 0")
+            CartProductListItem(v-for="item in cart",
+                :key="item.name",
+                :item="item")
+          .empty.has-text-centered(v-else-if="!total")
+            h3 Your cart is empty.
+            nuxt-link(exact, :to="getHomeURL")
+              button.button(@click="closeSidebarCart()") Fill er up!
+        div.cart-sidebar-footer(v-if="total > 0")
+          p
+            b Total: {{ amount }} {{ currency }}
+          nuxt-link(exact, :to="getCartURL")
+            .button.is-primary(@click="closeSidebarCart()") Checkout
+</template>
+
+<script>
+import { createNamespacedHelpers } from "vuex"
+import Switch from "@/components/Switch"
```

Comment:
I am not sure, but switch looks to be unused? Not sure why isn't eslint warning about it

Reply:
I'll do it tomorrow 😅 It's almost 00:30 in Spain! Good night!

Reply:
It's 01:30 in Belarus! Yeah let's finish this tomorrow! :D